### ISSUE TITLE
suders config not working on Ubuntu 21.10 

### ISSUE THREAD

**Issue by sterni1971** [sshuttle/sshuttle#724]
Hi!

strange bug after update to 1.1 the suders config on my ubuntu21.10. The sudoers
don't work. 

This don't work:
sshuttle --dns --exclude foo -v -r foo 123.123.0.0/16 

with sudoers config
"""
Cmnd_Alias SSHUTTLE89A = /usr/bin/env PYTHONPATH=/usr/local/lib/python3.9/dist-packages/sshuttle-1.1.0-py3.9.egg /usr/bin/python /usr/local/bin/sshuttle *

foo ALL=NOPASSWD: SSHUTTLE89A
""""

Command output
"""
c : Starting firewall manager with command: ['/usr/bin/env', 'PYTHONPATH=/usr/local/lib/python3.9/dist-packages/sshuttle-1.1.0-py3.9.egg', '/usr/bin/sudo', '-p', '[local sudo] Password: ', '/usr/bin/python', '/usr/local/bin/sshuttle', '-v', '--method', 'auto', '--firewall']
[local sudo] Password: 
""""

**Comment by brianmay** [comment]
Sounds like the same problem as reported in #723.

**Comment by seanschneeweiss** [comment]
Can confirm the bug on Ubuntu 20.04 and Python 3.8.10.
<sub>Sean Schneeweiss <sean.schneeweiss@daimler.com>, Daimler TSS GmbH, [Provider Information](https://github.com/mercedes-benz/daimler-foss/blob/master/PROVIDER_INFORMATION.md)</sub>

**Comment by seanschneeweiss** [comment]
Looks like the regression was introduced with https://github.com/sshuttle/sshuttle/pull/708/commits/d6f75fae255a9f7f4d306c0cd5e37ea5dc7b1c6f

Changing `/usr/bin/env`to `/usr/bin/sudo` seems to be a workaround.
https://github.com/sshuttle/sshuttle/blob/3d51bcba95a89b30d593a38b124b10278d2393c7/sshuttle/sudoers.py#L17
```diff
- Cmnd_Alias %(ca)s = /usr/bin/env PYTHONPATH=%(dist_packages)s %(py)s %(path)s * 
+ Cmnd_Alias %(ca)s = /usr/bin/sudo PYTHONPATH=%(dist_packages)s %(py)s %(path)s * 
```

**Comment by seanschneeweiss** [comment]
Could it be that the following has the wrong order:
https://github.com/sshuttle/sshuttle/blob/3d51bcba95a89b30d593a38b124b10278d2393c7/sshuttle/client.py#L228-L229
```py
sudo_cmd = sudo_cmd + pp_prefix
doas_cmd = doas_cmd + pp_prefix
```

**Comment by sterni1971** [comment]
I fixed it in the sudo config this way

```
# WARNING: When you allow a user to run sshuttle as root,
# they can then use sshuttle's --ssh-cmd option to run any
# command as root.

#Cmnd_Alias SSHUTTLE89A = /usr/bin/env PYTHONPATH=/usr/local/lib/python3.9/dist-packages/sshuttle-1.1.0-py3.9.egg /usr/bin/python /usr/local/bin/sshuttle *

Cmnd_Alias SSHUTTLE89A = /usr/bin/python /usr/local/bin/sshuttle * 
Defaults env_keep += "PYTHONPATH=/usr/local/lib/python3.9/dist-packages/sshuttle-1.1.0-py3.9.egg"

foo ALL=NOPASSWD: SSHUTTLE89A
```

**Comment by skuhl** [comment]
That commit appears to be the problem. I didn't mean to change the order of the command around with it. I'm preparing a fix.

**Comment by skuhl** [comment]
This should be fixed by merge #727.
